libs/utils/tests/logger_plain_test.rs

```diff
@@ -0,0 +1,43 @@
+use std::fs::{File, OpenOptions};
+use std::io::{self, BufRead};
+use tempfile::*;
+use tracing::*;
+use utils::logging::*;
+
+fn read_lines(file: File) -> io::Lines<io::BufReader<File>> {
+    io::BufReader::new(file).lines()
+}
+
+#[test]
+fn test_plain_format_has_message_and_custom_field() {
+    std::env::set_var("RUST_LOG", "warn");
+
+    let tmp_dir = TempDir::new().unwrap();
+    let log_file_path = tmp_dir.path().join("logfile");
+
+    let log_format = LogFormat::Plain;
+    let _log_file = init(&log_file_path, true, log_format).unwrap();
+    assert!(log_file_path.exists());
+
+    let custom_field: &str = "hi";
+    trace!(custom = %custom_field, "test log message");
+    debug!(custom = %custom_field, "test log message");
+    info!(custom = %custom_field, "test log message");
+    warn!(custom = %custom_field, "test log message");
+    error!(custom = %custom_field, "test log message");
+
+    let log_file = OpenOptions::new().read(true).open(&log_file_path).unwrap();
+
+    let lines = read_lines(log_file);
+    for line in lines {
+        let content = line.unwrap();
+        let json = serde_json::from_str::<serde_json::Value>(&content);
+        assert!(!json.is_ok());
```

Comment:
```suggestion
        serde_json::from_str::<serde_json::Value>(&content).unwrap_err();
```

This would handle the clippy error and simplify, since the return value was never needed.

Reply:
:+1: